src/map/pet.cpp

```diff
@@ -1148,6 +1151,11 @@ int pet_select_egg(struct map_session_data *sd,short egg_index)
 	if(egg_index < 0 || egg_index >= MAX_INVENTORY)
 		return 0; //Forged packet!
 
+	if(sd->trade_partner){	//The player have trade in progress.
+		clif_displaymessage(sd->fd, "You need to complete the trade first.");
```

Comment:
Please, do not hardcode messages, move it to map_msg.conf

Reply:
Thinking about it , it's better to keep this information from the player , i think it's better to remove the message.